lib/config/sass-lint.yml

```diff
@@ -25,6 +25,7 @@ rules:
   no-css-comments: 1
   no-debug: 1
   no-disallowed-properties: 0
+  no-domains: 0
```

Comment:
I think we should enable this by default, even though it will lead to a duplication in error messages it may force some people to check the docs to disable the extra option you've added in `no-url-protocols`